frontend/sw360-portlet/src/main/resources/META-INF/resources/html/utils/includes/searchReleases.jsp

```diff
@@ -84,8 +92,12 @@
         var homeUrl = themeDisplay.getURLHome().replace(/\/web\//, '/group/');
 
         $('#addLinkedReleasesToReleaseButton').on('click', showReleaseDialog);
-        $('#searchbuttonrelease').on('click', function() {
-            releaseContentFromAjax('<%=PortalConstants.RELEASE_SEARCH%>', $('#searchrelease').val(), function(data) {
+        $('#searchbuttonrelease').on('click', function(){
+            var searchRelease = $('#searchrelease').val();
+            if($('#exactMatchRelease').is(':checked') && !(searchRelease.startsWith("\"") && searchRelease.endsWith("\""))){
```

Comment:
please correct the indentation.
`function() {` at line `95` and `if ($('#exactMatchRelease').is(':checked') && !(searchRelease.startsWith("\"") && searchRelease.endsWith("\""))) {`

also at line `48`.


please do the same change in `searchProjects.jsp` file as well.